Adjustments to installation instructions (#152)
hydrologie/xhydro: CONTRIBUTING.rst

        git commit -m "Your detailed description of your changes."
        git push origin name-of-your-bugfix-or-feature

>     If ``pre-commit`` hooks fail, try fixing the issues, re-adding the files, and re-committing your changes (or, if need be, you can skip them with `git commit --no-verify`).

Reviewer: ```suggestion
    If ``pre-commit`` hooks fail, try fixing the issues, re-staging the files to be committed, and re-committing your changes (or, if need be, you can skip them with `git commit --no-verify`).
```

        # To simply test that the docs pass build checks
        python -m tox -e docs

> #. If changes to your branch are made on Github, you can update your local branch with:

Reviewer: ```suggestion
#. If changes to your branch are made on GitHub, you can update your local branch with:
```

    .. code-block:: console

        git checkout name-of-your-bugfix-or-feature
        git fetch
        git pull origin name-of-your-bugfix-or-feature

>     If you have conflicts, you might need to replace `git pull` with `git merge` and resolve the conflicts manually.

Reviewer: ```suggestion
    If you have merge conflicts, you might need to replace `git pull` with `git merge` and resolve the conflicts manually.
```

    Resolving conflicts from the command line can be tricky. If you are not comfortable with this, you can ignore the last command and instead use a GUI like PyCharm or Visual Studio Code to merge the remote changes and resolve the conflicts.

#. Before merging, your Pull Request will need to be based on the `main` branch of the `xhydro` repository. If your branch is not up-to-date with the `main` branch, you can perform the same steps as above to update your branch, replacing `name-of-your-bugfix-or-feature` with `main`.

> #. To prevent unnecessary testing of branches that are not ready for review, the `xhydro` repository is set up to run tests only when a Pull Request has been "approved" by a maintainer. Similarly, the notebooks within documentation will only be rebuilt when the Pull Request is "approved", or if the Pull Request makes explicit changes to them. As such, additional changes to the Pull Request might required once a maintainer approved the Pull Request to ensure that the tests pass and the documentation can be built.

Reviewer: ```suggestion
#. To prevent unnecessary testing of branches that are not ready for review, the `xhydro` repository is set up to run tests only when a Pull Request has been "approved" by a maintainer. Similarly, the notebooks within documentation will only be rebuilt when the Pull Request is "approved", or if the Pull Request makes explicit changes to them. As such, additional changes to the Pull Request might be required after the Pull Request is approved to ensure that the tests pass and the documentation can be built.
```

#. The pull request should include tests and should aim to provide `code coverage <https://en.wikipedia.org/wiki/Code_coverage>`_ for all new lines of code. You can use the `--cov-report html --cov xhydro` flags during the call to ``pytest`` to generate an HTML report and analyse the current test coverage.

#. All functions should be documented with `docstrings` following the `numpydoc <https://numpydoc.readthedocs.io/en/latest/format.html>`_ format.

> #. If the pull request adds functionality, either update the documentation or create a new notebook in the `docs/notebooks` directory that demonstrates the feature. the Jupyter notebooks located in the documentation should also be updated. Library-defining features should also be listed in ``README.rst``.

Reviewer: ```suggestion
#. If the pull request adds functionality, either update the documentation or create a new notebook in the `docs/notebooks` directory that demonstrates the feature. The Jupyter notebooks located in the documentation should also be updated. Library-defining features should also be listed in ``README.rst``.
```

> #. The ChangeLog should be updated with a brief description of the changes made in the pull request. If this is your first contribution to the project, please add your name to the `AUTHORS.rst` and `.zenodo.json` files.

Reviewer: ```suggestion
#. The ChangeLog should be updated with a brief description of the changes made in the Pull Request. If this is your first contribution to the project, please add your name and information to the `AUTHORS.rst` and `.zenodo.json` files.
```

> #. The pull request should work for all currently supported Python versions. Check the `pyproject.toml` or `tox.ini` files for the supported versions.

Reviewer: This is better than the current writing. Definitely to be pushed up to the template.
